dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -151,15 +152,15 @@ private static ArtifactResults generateReport(Configuration configuration, Path
       DependencyGraph transitiveDependencies) {
     Map<String, String> expectedVersionMap = graph.getHighestVersionMap();
     Map<String, String> actualVersionMap = transitiveDependencies.getHighestVersionMap();
-    
+
```

Comment:
Please set your IDE not to muck with existing trailing white space. Internal tools work a bit better than github here, but official policy is not to change this and not to comment on it.

Reply:
Got it.